### ISSUE TITLE
怎么进行模型测试

### ISSUE THREAD

**Issue by 0xFbing** [kingo233/FCT-Pytorch#8]
请问作者这个测试代码在哪，我自己写了一个测试代码不知道哪里弄错了，结果非常低
Testing DataLoader 0: 100%|██████████| 11/11 [00:07<00:00,  1.54it/s]
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       Test metric             DataLoader 0
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
    dice/LV_test_dice     5.5974485463350376e-11
   dice/MYO_test_dice      5.878756509369865e-11
    dice/RV_test_dice      4.758759011846969e-11
   dice/all_test_dice      5.411654804832189e-11
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
tensor(7.3430e-11, device='cuda:1')
